### PR TITLE
Add BazelRemoteAPI module

### PR DESCRIPTION
**Pull request by dmbryson**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -6,23 +6,74 @@
...
+	if [ ! -d ExternalRepositories/remote-apis ]; then \
+		git -C ExternalRepositories clone --depth 1 https://github.com/bazelbuild/remote-apis; \
+	else \
+		git -C ExternalRepositories/remote-apis pull --rebase; \
+	fi
+	if [ ! -d ExternalRepositories/googleapis ]; then \
+		git -C ExternalRepositories clone --depth 1 https://github.com/googleapis/googleapis; \
+	else \
+		git -C ExternalRepositories/googleapis pull --rebase; \
+	fi
```
Comment (ddunbar) [inline Makefile:28]: Should we just use submodules, or do we want to keep this only in the Makefile?

Comment (dmbryson) [reply Makefile:28]: Good question.   I had started down the submodule route, but that does mean that everyone that references llbuild2 will clone those even when the are only used during generation.

File: Package.resolved [hunk 2]
```diff
@@ -1,6 +1,33 @@
 {
   "object": {
     "pins": [
+      {
+        "package": "Commander",
```
Comment (aciidb0mb3r) [inline Package.resolved:5]: where are we getting these from?

Comment (dmbryson) [reply Package.resolved:5]: I think because I referenced an old swift-grpc here.   Trying to fix up the merge conflicts, but git is fighting me.  Sigh.

**Pull request merged**